### ISSUE TITLE
Flipper still crash with TAMA update. Null pointer derefence. 

### ISSUE THREAD

**Issue by Mika73** [RogueMaster/flipperzero-firmware-wPlugins#180]
### Describe the bug.

Flipper does still crash with latest Tama update. When try read Sub-Ghz. 

### Reproduction

Still crash with Sub-Ghz read.

**Comment by Mika73** [comment]
[Fiat.sub.zip](https://github.com/RogueMaster/flipperzero-firmware-wPlugins/files/9267086/Fiat.sub.zip)

Send/Emulate this totally crash Flipper. I managed read file with hopping mode.

**Comment by Mika73** [comment]
Also downloaded new setting_user file but it didnt help crash

**Comment by thespawnat** [comment]
I'm able to capture signals in "read mode" with "detect raw = on" on 433,92 and can send them directly without crash (no saving, direct send). maybe I can test with 300/312 in the next days to check that frequencies too.

I noticed an issue with 868 MHz in the new Read-Mode (separate issue created), as my car-key-signals are only detected in explicit raw-mode (read-raw) . sending from read-raw results didn't cause a reboot for me, even on the last releases, only in the new "read".

**Comment by zviratko** [comment]
Also crashing on TAMA. You can "workaround" this by rebooting without the SD card and then inserting it. It will then work fine, somehow. I reformatted my SD card and did a clean install, still crashes, so it's a broken release...

**Comment by RogueMaster** [comment]
@zviratko you may be missing the assets for SD from the release. Specifically the dolphin/manifest.txt file or subghz/assets/extended_range.txt

**Comment by RogueMaster** [comment]
@Mika73 I can recreate the crash but only with the file you sent which seems to be specific to a special protocol Keeloq 64bit at 433.92AM. will check further

**Comment by zviratko** [comment]
Nope, those are all here. I did a clean install and all resources are present.

**Comment by credek** [comment]
My flipper also have this problem, crash and rebooting when i want to read a signal.

**Comment by RogueMaster** [comment]
@credek please provide more information to recreate. Please also see if your problem occurs on the unleashed build. @Mika73 your file also crashes unleashed. Must have something to do with the keeloq updates but unsure

**Comment by credek** [comment]
@RogueMaster I trying to read my key fob with 'Read' option. When I click a button on my remote Flipper rebooting and i get this:
![crash](https://user-images.githubusercontent.com/55182807/183129630-8ffb9a6e-8fe1-4bd5-a841-ea84b17bce10.jpg)

**Comment by RogueMaster** [comment]
Please check if unleashed release gives you the same issue

**Comment by credek** [comment]
@RogueMaster no issue on unleashed and 0.63.4-0805-RM-NOFAP release, just on this last one.

**Comment by RogueMaster** [comment]
@credek Please take picture of the last page of your about; it should be the same one that states "0.63.4"

**Comment by credek** [comment]
@RogueMaster https://streamable.com/6sb925
Everytime on the latest release :(
![TAMA](https://user-images.githubusercontent.com/55182807/183141788-b103504a-7b63-43cd-b47d-981286b37e49.jpg)

**Comment by jfurmancom** [comment]
I can confirm this. Keeloq definitely crashing Flipper. This issue not happens on clean "Unleashed" firmware.

**Comment by juda73** [comment]
Same crash with SCANAP in  WIFI (MARAUDER)

**Comment by zviratko** [comment]
I realize that I'm probably facing a different (at least in flavor) issue, but I want to note that even if I boot without an SD card and then reinsert it, flipper still crashes later (few minutes, few hours, doing nothing it just reboots and crash loops with NULL pointer deference) also just noticed that it actually boots and crashes only after about 5 seconds on Home Screen.

**Comment by RogueMaster** [comment]
Try now and re-open if you still have issues
